tests/e2e/run-scanner-v4.bats

```diff
@@ -615,13 +615,13 @@ verify_no_scannerV4_deployed() {
 
 verify_no_scannerV4_indexer_deployed() {
     local namespace=${1:-stackrox}
-    run kubectl -n "$namespace" get deployments -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
+    run "${ORCH_CMD}" -n "$namespace" get deployments -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
```

Comment:
I have no experience with BATS, but please note that the script is not exactly a drop-in replacement for bare `kubectl`. In particular:

> NOTE: Reads all of stdin. If no input should be passed to kubectl, caller should redirect stdin from /dev/null!

Same below.

Reply:
Thanks for the reminder. I have verified that bats doesn't fiddle around with stdin.
That means, when writing

```
run cmd
```

the semantics of reading `stdin` are unchanged for `cmd`, regardless of this thing running in a bats context or not.

That being said, it seems to work in CI. But according to my tests it wouldn't work in an interactive shell. Hence I have added the `</dev/null`.